pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -989,26 +1163,62 @@ def __init__(
         if not control_values:
             control_values = "1" * len(control_wires)
 
-        control_int = self._parse_control_values(control_wires, control_values)
         self.control_values = control_values
-
-        self._padding_left = control_int * 2
-        self._padding_right = 2 ** len(wires) - 2 - self._padding_left
-        self._CX = None
+        self.control_int = self._parse_control_values(self.control_wires, self.control_values)
+        self._cx = None
 
         super().__init__(*params, wires=wires, do_queue=do_queue)
 
     @property
     def num_params(self):
         return 0
 
-    def _matrix(self, *params):
-        if self._CX is None:
-            self._CX = block_diag(
-                np.eye(self._padding_left), PauliX.matrix, np.eye(self._padding_right)
-            )
+    @staticmethod
+    def compute_matrix(control_int, n_wires):
+        """Canonical matrix representation of the MultiControlledX operator.
+
+        **Examples:**
```

Comment:
```suggestion
        **Examples**
```
(haven't been noticing, so not sure if the other examples above also have the colon!)

Reply:
By now I should have learnt it, this was an outlier I hope